src/main/resources/application.conf

```diff
@@ -5,8 +5,10 @@ stripe-scala {
   file-upload-endpoint="https://uploads.stripe.com"
   file-upload-endpoint=${?STRIPE_FILE_UPLOAD_ENDPOINT}
 
-  apiKey = ${STRIPE_API_KEY}
+  api-key = ${STRIPE_API_KEY}
 
-  numberOfRetries=5
-  numberOfRetries=${?STRIPE_NUMBER_OF_RETRIES}
+  number-of-retries=5
+  number-of-retries=${?STRIPE_NUMBER_OF_RETRIES}
+
+  file-upload-chunk-timeout="5 minutes"
```

Comment:
No idea, theoretically you should never hit this timeout, its only the timeout for reading the input stream. Do you have a better idea for a timeout? Else I was thinking of leaving it and if it causes a problem I can always change it later